src/hub/dataload/sources/drugcentral/drugcentral_dump.py

```diff
@@ -1,15 +1,134 @@
+import csv
 import os
 
 import biothings
+import psycopg2
+
 import config
 
 biothings.config_for_app(config)
 
-from config import DATA_ARCHIVE_ROOT
-from biothings.hub.dataload.dumper import DummyDumper
+from biothings.hub.dataload.dumper import BaseDumper
 
+from config import DATA_ARCHIVE_ROOT, DRUGCENTRAL_PASSWORD
 
-class DrugCentralDumper(DummyDumper):
 
+class DrugCentralDumper(BaseDumper):
     SRC_NAME = "drugcentral"
     SRC_ROOT_FOLDER = os.path.join(DATA_ARCHIVE_ROOT, SRC_NAME)
+
+    # More info about the public DrugCentral Postgres database https://drugcentral.org/download
+    HOST = "unmtid-dbs.net"
+    PORT = 5433
+    DATABASE = "drugcentral"
+    USER = "drugman"
+
+    def prepare_client(self):
+        # Connect to the database
+        self.client = psycopg2.connect(
+            database=self.DATABASE,
+            user=self.USER,
+            password=DRUGCENTRAL_PASSWORD,
+            host=self.HOST,
+            port=self.PORT,
+        )
+        # Create a cursor
+        self._cursor = self.client.cursor()
+
+    @property
+    def cursor(self):
+        # Create a cursor if it doesn't exist
+        if not getattr(self, "_cursor", None):
+            if self.client:
+                self._cursor = self.client.cursor()
+            else:
+                self.logger.debug(
```

Comment:
I think here, it's at least a `warning` log instead of `debug`.

Reply:
Updated in most recent commit f48f51b
```               
                self.logger.warning(
                    "_cursor was not initialized and client is not active."
                )
```